### ISSUE TITLE
Using as third part library - INSTALL FAILED

### ISSUE THREAD

**Issue by ryuuji06** [openMVG/openMVG#1318]
Hello, I'm in ubuntu, and I want to use OpenMVG as a third part library dependency in Cmake, but it's failling at make install. I followed the steps in BUILD.md, but I'm not sure I understood it right. I've done the following, all at openMVG_build dir:

 1) cmake, adding tests and install (done successfully)

 `$ cmake -DCMAKE_BUILD_TYPE=RELEASE -DOpenMVG_BUILD_TESTS=ON -DCMAKE_INSTALL_PREFIX:STRING="/home/program-files/openMVG_build/install" . ../openMVG/src/`

 2) make (done successfully)

 3) make test (successfull)

 4) make install

 Failed, I got the following error:

Install the project...
-- Install configuration: "RELEASE"
-- Installing: /home/program-files/openMVG_build/install/include/openMVG_dependencies
CMake Error at cmake_install.cmake:41 (file):
  file INSTALL cannot make directory
  "/home/program-files/openMVG_build/install/include/openMVG_dependencies":
  No such file or directory

**Comment by pmoulon** [comment]
1. Do you have the rights to write in this folder?
- Does the directory exits `/home/program-files/openMVG_build/install`?

2. Else you can try to install OpenMVG on your computer without specifying the path:
```
$ cmake -DCMAKE_BUILD_TYPE=RELEASE -DOpenMVG_BUILD_TESTS=ON . ../openMVG/src/
$ make -j
$ make test
$ make install
```

**Comment by ryuuji06** [comment]
Well, I guess I have the rights, I created it myself and have made others builds and installs in the same directory (\home\program-files).

Actually the directory .../install doesn't exist, I thought it was supposed to be created in the building. Then I manually created such directory, but it seems the make install expected some files inside of it, then I guess it had to be created during the building, but it wasn't.
